### PR TITLE
Add Python 3.12 support

### PR DESCRIPTION
**Pull request by pfouque**
Just adding the last version of Python 3.12 to the test matrix and package classifier

Extracted from #283, this PR depends on https://github.com/yourlabs/django-dbdiff/pull/21

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -25,7 +29,8 @@ deps =
 deps =
     # sphinx
     Sphinx==4.2.0
-    django-dbdiff
+    ; django-dbdiff
+    git+https://github.com/pfouque/django-dbdiff.git@fix312#egg=django-dbdiff
```
Comment (pfouque) [inline tox.ini:33]: @marianoeramirez you're now hooked to my fork of django-dbdiff ... :/

Comment (marianoeramirez) [reply tox.ini:33]: Yes, but we will have to wait a long time before them review and integrate the PR. Once that happen I will update the project.

Comment (pfouque) [reply tox.ini:33]: Ok, I'm fine with that, I wanted to be sure you noticed! ;) 

In the meantime, I started a branch to remove this dependency, 
and use load_fixtures/dump_fixtures instead + a backport of the assertNoDiff statement
It was promising, and only the command was missing tests.

Do you want me to publish the draft?

Comment (pfouque) [reply tox.ini:33]: https://github.com/yourlabs/django-cities-light/pull/288

**Pull request merged**